src/roles/webfilter-deploy/tasks/main.yml

```diff
@@ -59,11 +59,18 @@
   - name: "Ensure application properties contains webfilter"
```

Comment:
Change name to something task appropriate. Looks like this task has same name as previous.